plugins/outputs/dynatrace/README.md

```diff
@@ -1,53 +1,120 @@
 # Dynatrace Output Plugin
 
-This plugin is sending telegraf metrics to [Dynatrace](https://www.dynatrace.com). It has two operational modes.
+This plugin sends Telegraf metrics to [Dynatrace](https://www.dynatrace.com) via the [Dynatrace Metrics API V2](https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/). It may be run alongside the Dynatrace OneAgent for automatic authentication or it may be run standalone on a host without a OneAgent by specifying a URL and API Token.
 
-Telegraf minimum version: Telegraf 1.16 
-Plugin minimum tested version: 1.16
+## Requirements
+
+You will either need a Dynatrace OneAgent (version 1.201 or higher) installed on the same host as Telegraf; or a Dynatrace environment with version 1.202 or higher. Monotonic counters (e.g. diskio.reads, system.uptime) require Dynatrace 208 or later.
+
+- Telegraf minimum version: Telegraf 1.16
+- Plugin minimum tested version: 1.16
+
+## Getting Started
 
-## Running alongside Dynatrace OneAgent
+Setting up Telegraf is explained in the [Telegraf Documentation](https://docs.influxdata.com/telegraf/latest/introduction/getting-started/).
+The Dynatrace exporter may be enabled by adding an `[[outputs.dynatrace]]` section to your `telegraf.conf`.
+All configurations are optional, but if a `url` is specified then an `api_token` is required.
+To see all available options, see [Configuration](#configuration) below.
 
-if you run the Telegraf agent on a host or VM that is monitored by the Dynatrace OneAgent then you only need to enable the plugin but need no further configuration. The Dynatrace telegraf output plugin will send all metrics to the OneAgent which will use its secure and load balanced connection to send the metrics to your Dynatrace SaaS or Managed environment.
+### Running alongside Dynatrace OneAgent
+
+If you run the Telegraf agent on a host or VM that is monitored by the Dynatrace OneAgent then you only need to enable the plugin, but need no further configuration. The Dynatrace telegraf output plugin will send all metrics to the OneAgent which will use its secure and load balanced connection to send the metrics to your Dynatrace SaaS or Managed environment.
+
+note: The name and identifier of the Telegraf host will be added as a dimension to every metric. If this is undesireable, then the output plugin may be used in standalone mode using the directions below.
+
+```toml
+[[outputs.dynatrace]]
+  ## No options are required. By default, metrics will be exported via the OneAgent on the local host.
```

Comment:
Is the default export interval fine for us?

Reply:
The flush interval is not set per-output so we don't have a lot of control there. We could mention a recommended interval if we want. Having a hard time finding a documented default flush interval but it appears to be 10s according to some random internet posts.

Reply:
Mentioning a recommended one sounds good. What did we say in the other exporters? I think it was 10-60s.